src/sentry/static/sentry/app/components/events/relatedEvents/emptyState.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {tct} from 'app/locale';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import {Panel} from 'app/components/panels';
+import {DEFAULT_RELATIVE_PERIODS} from 'app/constants';
+
+type Props = {
+  message?: string;
+};
+
+const EmptyState = ({message}: Props) => (
+  <Panel>
+    <EmptyStateWarning small>
+      {message ??
+        tct('No related events have been found for the [timePeriod].', {
```

Comment:
I think this could be rephrased as "No related events have been found 12 hours either before and after the occurrence of this event."

Reply:
ok..it's fine for me :)